rust-connectors/sinks/sql/README.md

```diff
@@ -0,0 +1,98 @@
+# Fluvio SQL Sink connector
+The SQL Sink connector reads records from Fluvio topic, applies configured transformations, and 
+sends new records to the SQL database.
+## Supported databases
+1. PostgreSQL
+2. SQLite
+
+### Data types
+| Model           | PostgreSQL                   | SQLite       |                                          
+|:----------------|:-----------------------------|:-------------|
+| Bool            | BOOL                         | BOOLEAN      |
+| Char            | CHAR                         | INTEGER      |
+| SmallInt        | SMALLINT, SMALLSERIAL, INT2  | INTEGER      |
+| Int             | INT, SERIAL, INT4            | INTEGER      |
+| BigInt          | BIGINT, BIGSERIAL, INT8      | BIGINT, INT8 |
+| Float           | REAL, FLOAT4                 | REAL         |
+| DoublePrecision | DOUBLE PRECISION, FLOAT8     | REAL         |
+| Text            | VARCHAR, CHAR(N), TEXT, NAME | TEXT         |
+| Bytes           | BYTEA                        | BLOB         |
+| Numeric         | NUMERIC                      | REAL         |
+| Timestamp       | TIMESTAMP                    | DATETIME     |
+| Date            | DATE                         | DATE         |
+| Time            | TIME                         | TIME         |
+| Uuid            | UUID                         | BLOB, TEXT   |
+| Json            | JSON, JSONB                  | TEXT         |
+
+
+## Transformations
+The SQL Sink connector expects the data in [Fluvio SQL Model](../../models/fluvio-model-sql) in JSON format.
+In order to work with different data formats or data structures, transformations can be applied.
+The transformation is a SmartModule pulled from the SmartModule Hub. Transformations are chained according to the order
+in the config. If a SmartModule requires configuration, it is passed via `with` section of `transforms` entry. 
+## Usage
+
+### Configuration via parameters
+| Option       | default                     | type   | description                                           |
+|:-------------|:----------------------------| :---   |:------------------------------------------------------|
+| hub_url      | http://127.0.0.1:8080       | String | The paramaters key for the smartmodule hub url        |
```

Comment:
not sure if we should use this default, maybe we could make this a required parameter for now just like database_url until the public hub_url is officially announced

Reply:
There is a small typo here and on the next line

`paramaters` -> `parameters`

Reply:
Done